Analyse_Birds.qmd

```diff
@@ -457,7 +455,268 @@ Here we have **58%** of good predictions, which is somewhat below the common thr
 This result is not surprising given our initial data. Because they are from different studies some variables barely contains some species or years, etc... This could explain why our model is not suitable. 
 
 
-# II. ANOVA
+# II. Analysing the relationship between nest fate and county, habitat, species, year, initiation date using a generalized linear model (GLM) with a Poisson distribution
+
+## Variables presentation:
+
+```{r poisson_data_conversion}
+data$Nest_Fate <- as.factor(data$Nest_Fate)
+data$County <- as.factor(data$County)
+data$Habitat <- as.factor(data$Habitat)
+data$Year <- as.factor(data$Year)
+```
+
+
+-   *County* : the county in the state where the state has been found.
+    14 levels factor.
+-   *Habitat* : the type of habitat in which the nest has been found. 4
+    levels factor.
+-   *Species* : the species of the bird that has build the nest. 8
+    levels factor.
+-   *Year* : the year of the nest building. 21 levels factor.
+-   *Number_Fledged* : the number of birds that hatch on the nest when
+    they hatch. Quantitative variable.
+
+We want to study the impact of four qualitative variables, the habitat, the county, the species and the year over the number of young hatched in the nest *"Number_Fledged"*. It is a counting variable, which imply that it follows a Poisson distribution. A negative binomial distribution could be used but is preferred when the count values are high (which is not really the case here since the highest value is of 6)
+
+A Poisson law is used to describe rare, discrete and independent events. Hence the variable $Y$ that follows a Poisson distribution of parameter $\lambda$ is written:
+$$Pr(Y=y)=\frac{e^{-\lambda}.\lambda^y}{y!}$$
+where $y$ represent the value of the count (here $y$ ∈ [0-6]) and $\lambda$ is the mean value and variance of the Poisson distribution (because in a Poisson law E($y$)= Var($y$)=$\lambda$).
+
+
+## A. Data exploration
+
+Before any analysis, it is important to have a look about the data.
+
+### Outlayer values in the response variable $Y$ (the number of young in the nest, Number_Fledged) and values distribution:
+
+```{r poisson_datahist, include=TRUE, fig.height=5, fig.width=5}
+par(mfrow=c(2,2))
+# Boxplot
+boxplot(data$Number_Fledged,col='dodgerblue3',ylab='Number of young in nest')
+# Cleveland plot
+dotchart(data$Number_Fledged,pch=16,col='dodgerblue3',xlab='Number of young in nest')
+# Histogram
+hist(data$Number_Fledged,col='dodgerblue3',xlab="Number of young in nest",main="")
+# Quantile-Quantile plot
+qqnorm(data$Number_Fledged,pch=16,col='dodgerblue3',xlab='')
+qqline(data$Number_Fledged,col='red',lwd=2)
+```
+
+There isn't any outlayer values. There are a lot of value at 0. Maybe we can study only the number of young when there are some. There could be a normal distribution.
+
```

Comment:
I think it is "there ARE NOT" and not "isn't"
An i am not sure if it is "there could be" maybe it would be better with "it could" but i am not sure